app/org/sagebionetworks/bridge/play/controllers/ParticipantController.java

```diff
@@ -115,12 +120,6 @@ public Result updateParticipant(String userId) {
             throw new BadRequestException("ID in JSON does not match email in URL.");
         }
         participantService.updateParticipant(study, session.getParticipant().getRoles(), userId, participant);
-        
-        // Push changes to the user's session, including consent statuses.
-        CriteriaContext context = new CriteriaContext.Builder()
-                .withStudyIdentifier(study.getStudyIdentifier()).build();
-        session = authenticationService.updateSession(study, context, userId);
-        updateSession(session);
```

Comment:
We could automate this. Currently though there's a button and an API to do it as a separate step. If you sign people out, you have to do work on the client to determine if this was an edit of the current user's session, etc, which is error-prone.